frontend/gatsby-config.js

```diff
@@ -32,6 +32,8 @@ module.exports = {
         background_color: "#663399",
         theme_color: "#663399",
         display: "minimal-ui",
+
```

Comment:
Sorry, but could you please remove this space 🙏

Reply:
Done.

Best regards.